calcom/cal.com
# Tandem Video

## What does this PR do?

This adds support for hosting meetings on Tandem.

## Type of change


## How should this be tested?

I tested connecting the integration, authorizing a user, and scheduling meetings against our internal servers. 

This code should be ready for soft launch.  We'll want to do a round of testing once you are ready to host.  

To test, you'll just need a free account on Tandem, then go to one of your event types and add Tandem Video and schedule en event.  It should add a tandem link, and add events visible in Tandem, and it should also update correctly when you reschedule or cancel.  

## Checklist:
